From 803d24d3863156d5a76c46ee0ec06fa95626988b Mon Sep 17 00:00:00 2001
From: Marcelo Duarte Trevisani <marceloduartetrevisani@gmail.com>
Date: Thu, 21 Oct 2021 12:38:40 +0100
Subject: [PATCH] Fix setup for windows

---
 setup.py | 16 ++++------------
 1 file changed, 4 insertions(+), 12 deletions(-)

diff --git a/setup.py b/setup.py
index 9e1d337..783b4fe 100755
--- a/setup.py
+++ b/setup.py
@@ -1,13 +1,10 @@
-#!/usr/bin/env python
-
 import os
 from setuptools import setup, find_packages
 from distutils.core import Extension
 import platform
 
-work_dir = os.path.dirname(os.path.realpath(__file__))
-mod_dir = os.path.join(work_dir, 'src', 'confluent_kafka')
-ext_dir = os.path.join(mod_dir, 'src')
+
+ext_dir = os.path.join('src', 'confluent_kafka', 'src')
 
 INSTALL_REQUIRES = [
     'futures;python_version<"3.2"',
@@ -37,12 +34,7 @@ JSON_REQUIRES = ['pyrsistent==0.16.1;python_version<"3.0"',
 
 PROTO_REQUIRES = ['protobuf'] + SCHEMA_REGISTRY_REQUIRES
 
-# On Un*x the library is linked as -lrdkafka,
-# while on windows we need the full librdkafka name.
-if platform.system() == 'Windows':
-    librdkafka_libname = 'librdkafka'
-else:
-    librdkafka_libname = 'rdkafka'
+librdkafka_libname = 'rdkafka'
 
 module = Extension('confluent_kafka.cimpl',
                    libraries=[librdkafka_libname],
@@ -84,7 +76,7 @@ setup(name='confluent-kafka',
       ext_modules=[module],
       packages=find_packages('src'),
       package_dir={'': 'src'},
-      data_files=[('', [os.path.join(work_dir, 'LICENSE.txt')])],
+      data_files=[('', ['LICENSE.txt'])],
       install_requires=INSTALL_REQUIRES,
       classifiers=trove_classifiers,
       extras_require={
-- 
2.30.2
